aks/README.md

```diff
@@ -6,19 +6,21 @@
  export ARM_TENANT_ID=<Tenant Id>
  export ARM_SUBSCRIPTION_ID=<Subscription id>
  ```
- These env vars are used by the Terraform AzureRM provider to login to Azure with the configured SP credentials. Note the SP must have sufficient privileges to deploy resources in the subscription and must be created with `az ad sp create-for-rbac`, it cannot be created via the portal.
+ These env vars are used by the Terraform AzureRM provider to login to Azure with the configured SP credentials. 
+ Note the SP must have sufficient privileges, the created account must have at least contributor role on the tenant you may used az cli by running  `az ad sp create-for-rbac`or via the portal by creating app 
```

Comment:
You might want to make some minor fixes on the run-on sentences above.

Reply:
I am fixing it :)